### DISCUSSION TITLE
Navbar elements visible only to Guests cannot be seen by Guests.

### DISCUSSION THREAD

**Discussion by Meigs2** [requarks/wiki#3321] [Error / Bug Report]
**Describe the bug**
Assigning `guests` in `Visible to select groups` for a Navigation element does not actually display the element to guests.

**To Reproduce**
1. Go to 'Navigation' in the Administration Dashboard
2. Add an Navigation element
3. Set `Visible to select groups...` to `Guests`
4. Apply changes
5. View navbar from a Guest's view

**Expected behavior**
The navigation element should be visible.

**Screenshots**

**Host Info (please complete the following information):**
 - OS: Docker Container (Linux) on a Kubernetes cluster
 - Wiki.js version: 2.5.170
 - Database engine: postgres 12.2

**Additional context**
We would like some Navbar elements to not show when a user is logged in, however it seems Guests cannot view navbar elements assigned to them. We must use `Visible to everyone`.

**Comment by Valiantiam** [comment]
I was able to reproduce this as well.

Version: 2.5.268

**Comment by j007bond007** [comment]
This is happening to me as well on the latest version
